exercise/homework/startup_matchmaker/hw2/hw2.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <link rel="stylesheet" type="text/css" href="hw2.css">
+
+  <body>
+<div=class="begin">
+    <h1>Startup matchmaker</h1></div>
+    <div class ="bros">
+     <div class ="two-bros">
+      <div><b>Find developers</b></div>
+      <div> <b>Find designers</b></div>
+    </div>
+    <div class ="three-bros">
+    <div>How it works</div>
+    <div>Our team</div>
+    <div>Blog</div>
+  </div>
+</div>
+<hr class="new"></hr>
+<p><em>Because two brains <strike>taste</strike> are better than one.</em></p>
+<div class="body">
+  <center>
+<img src="../images/coworking.jpg" alt="coworking">
+</center>
+<div class="centered"><b>Meet your Match!</b><br>
+have a great idea for a<br> product,but need help<br>
+making it a reality? we're here<br>
+to help. Startup matchmaker<br>
+is the best place for designers<br>
+and developers to find each<br>
+other.
+<div class="button1"><button type="button" onclick="alert('hello')"> CREAT A PROFILE</button></div>
+</div>
+<div class="last">
+<div class="last1">
+<h2>create a Profile</h2>
+  <h4> are you a Designer?<br>
+    Developer? Put yourself<br> out there so that others<br> can find you!</h4>
+    <button type="button" onclick="alert('hello')"> SIGN UP NOW</button>
+</div>
+<div=class"last2">
```

Comment:
Hind you did right 👌 but you should put equal sign (=) in front of class.
ex: <div class="list 2">
Always the small mistake makes big trouble. Welcome to the programming 😂